### DISCUSSION TITLE
How to run the website locally

### DISCUSSION THREAD

**Discussion by AlexVCS** [lima-vm/lima#1843] [Q&A] [answered]
Hi I am working issue #1839 and have not been able to run the website locally. I did an `npm i` from the `website` directory, but when I run `npm run _serve`, `npm _hugo-dev`, etc. I get the following error:

`Error: failed to load modules: module "../_output" not found in "/Users/alexcurtisslep/Documents/projects/lima/website/_output"; either add it as a Hugo Module or store it in "/Users/alexcurtisslep/Documents/projects/lima/website/themes".: module does not exist`

I didn't see any documentation on how to do this, if you're able to steer me in the right direction that would be appreciated!

**Comment by balajiv113** [comment, marked as answer]
Hi,
Kindly run make docsy from root folder. That should fix.

Steps to run website are
- `make docsy` in root
- `npm i` in website folder
- `npm run serve` in website folder to view locally 

Will update the readme in sometime

**Reply by AlexVCS** [reply]
Thank you!
